src/nnet3/nnet-combine.cc

```diff
@@ -34,7 +34,7 @@ NnetCombiner::NnetCombiner(const NnetCombineConfig &config,
     nnet_params_(std::min(num_nnets, config_.max_effective_inputs),
                  NumParameters(first_nnet)),
     tot_input_weighting_(nnet_params_.NumRows()) {
-  SetDropoutProportion(0, &nnet_);
+  SetDropoutProportion(0, false, &nnet_);
```

Comment:
you can remove this 'false' argument from the function... just make it a fixed property of the component that can't be changed after you initialize.